src/applications/hca/helpers.jsx

```diff
@@ -20,6 +20,11 @@ import { isInMPI } from 'platform/user/selectors';
 
 import facilityLocator from '../facility-locator/manifest.json';
 
+export {
```

Comment:
@sanlouise anything that was importing the functions from this file will still work because of these lines.

Reply:
I actually don't know if this is a good approach or not. If reviewers object, I can find all the places that import the helpers from this file and update those imports.